common/src/main/java/hardcorequesting/common/client/interfaces/graphic/task/CheckBoxTaskGraphic.java

```diff
@@ -0,0 +1,41 @@
+package hardcorequesting.common.client.interfaces.graphic.task;
+
+import hardcorequesting.common.client.ClientChange;
+import hardcorequesting.common.client.interfaces.GuiQuestBook;
+import hardcorequesting.common.client.interfaces.widget.AbstractCheckBox;
+import hardcorequesting.common.client.interfaces.widget.LargeButton;
+import hardcorequesting.common.network.NetworkManager;
+import hardcorequesting.common.quests.task.CheckBoxTask;
+import net.minecraft.network.chat.TextComponent;
+
+import java.util.UUID;
+
+public class CheckBoxTaskGraphic extends TaskGraphic {
+    public CheckBoxTaskGraphic(CheckBoxTask task, UUID playerId, GuiQuestBook gui) {
+        super(playerId, gui, task);
+
+        addClickable(new AbstractCheckBox(gui, TextComponent.EMPTY, START_X, START_Y) {
+            @Override
+            public boolean getValue() {
+                return task.isToggled(playerId);
+            }
+
+            @Override
+            public void setValue(boolean val) {
+                task.setToggled(playerId, val);
+            }
+        });
+
+        addClickable(new LargeButton(gui, "hqm.quest.manualSubmit", 185, 200) {
```

Comment:
Is there a reason to need a manual submit button for this task? I'd otherwise imagine that the task would be completed with the press of the checkbox.